config/guestosconfig/known-modules.json

```diff
@@ -0,0 +1,69 @@
+[
```

Comment:
Why do we need this to be a separate file? Why not just configure this in `config/system-stats-monitor.json`?

And do we really need this to be configurable? Can we start with a hard coded list?

Reply:
having this configurable would be platform independent and it also helps the users on different OS.

Reply:
For context, this was hard-coded and Varsha made it a configurable list for maintenance reasons (as stated, for supporting different distros being one)

Reply:
But why do we need to reference to another config `known_modules.json` from an existing config `config/system-stats-monitor.json`?

Can't we just define this in `config/system-stats-monitor.json`?

Reply:
I prefer separate file. On different distros, we don't have to maintain 2 `system-stats-monitor` files (which will have duplicated config). Instead, we can just swap the `known_modules` config on the fs.

Reply:
adding to that it system-stats-monitor would become complex to maintain.

Reply:
Make sense to me.